File: src/main/java/calebzhou/rdi/microservice/model/json/weather/Alert.java

package calebzhou.rdi.microservice.model.json.weather;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class Alert {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("content")
    @Expose
    private List<Content> content = null;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public List<Content> getContent() {
        return content;
    }

    public void setContent(List<Content> content) {
        this.content = content;
    }

}

File: src/main/kotlin/calebzhou/rdi/microservice/model/json/weather/Alert.kt
package calebzhou.rdi.microservice.model.json.weather

import com.google.gson.annotations.SerializedName


data class Alert (

  @SerializedName("status"  ) var status  : String?            = null,
  @SerializedName("content" ) var content : ArrayList<Content> = arrayListOf()

)